android/java/org/chromium/chrome/browser/onboarding/P3aOnboardingActivity.java

```diff
@@ -49,7 +50,14 @@ protected void onCreate(Bundle savedInstanceState) {
         CheckBox p3aOnboardingCheckbox = findViewById(R.id.p3a_onboarding_checkbox);
         boolean isP3aEnabled = true;
         try {
-            isP3aEnabled = BravePrefServiceBridge.getInstance().getP3AEnabled();
+            // This is a hack for an unsolved JNI problem on android browsertests
```

Comment:
I think this is almost certainly because you just copied upstream targets and don't have the required deps for Brave

Reply:
I don't think so. the browser target is in dependency and I also tried included related files directly. They both didn't solve the problem.

Reply:
It could be that native libs are not loaded yet as @wchen342 mentioned. Usually we use that method to access native https://github.com/brave/brave-core/blob/master/android/java/org/chromium/chrome/browser/app/BraveActivity.java#L284

Reply:
seems like a bug in this class then and this code should be called in `finishNativeInitialization`?

Reply:
see `ChromeCachedFlags`